addons/product/models/product_pricelist.py

```diff
@@ -345,6 +345,15 @@ def get_import_templates(self):
             'label': _('Import Template for Pricelists'),
             'template': '/product/static/xls/product_pricelist.xls'
         }]
+    
+    def unlink(self):
+        for pricelist in self:
+            linked_items = self.env['product.pricelist.item'].sudo().with_context(active_test=False).search(
+                [('base', '=', 'pricelist'), ('pricelist_id', '=', pricelist.id)])
+            if linked_items:
+                raise UserError(_('You cannot delete this pricelist (%s), it is used in other(s) pricelist(s) : \n%s',
```

Comment:
```suggestion
                raise UserError(_('You cannot delete this pricelist (%s), it is used in other pricelist(s) : \n%s',
```
AFAIK, other doesn't require a 's' in this case.